### PR TITLE
Solution HW 2.3

### PR DESCRIPTION
**Pull request by Bogdan-Demianchuk**

### CODE REVIEW

File: src/main/java/core/basesyntax/Bulldozer.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer начал свою работу");
```
[the remark this answers is not in the document]
Comment (Bogdan-Demianchuk) [reply src/main/java/core/basesyntax/Bulldozer.java]: Done

File: src/main/java/core/basesyntax/MainApp.java [hunk 2]
```diff
@@ -12,5 +12,23 @@
...
+        Machine bulldozer = new Bulldozer();
+
+        bulldozer.doWork();
+        bulldozer.stopWork();
+
+        Machine track = new Track();
+
+        track.doWork();
+        track.stopWork();
+
```
[the remark this answers is not in the document]
Comment (Bogdan-Demianchuk) [reply src/main/java/core/basesyntax/MainApp.java]: Done

File: src/main/java/core/basesyntax/MainApp.java [hunk 3]
```diff
@@ -12,5 +12,16 @@
  * - Отправить все машини на работу и остановить работу всех машин.
  */
 public class MainApp {
+    public static void main(String[] args) {
+        Machine excavator = new Excavator();
+        excavator.doWork();
+        excavator.stopWork();
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/MainApp.java:18]: first you should send all machines to work and then stop them

Comment (Bogdan-Demianchuk) [reply src/main/java/core/basesyntax/MainApp.java]: Done

**Pull request closed without merging**